arch/risc-v/src/common/supervisor/riscv_sbi.c

```diff
@@ -63,23 +65,34 @@ static inline uintptr_t sbi_ecall(unsigned int extid, unsigned int fid,
      : "memory"
      );
 
-  return r1;
+  ret.error = r0;
+  ret.error = (uintreg_t)r1;
```

Comment:
why assign twice

Reply:
It is my fault, the second should be value.